dockerfiles/dataset/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+MAINTAINER Madhuri Yechuri <madhuri.yechuri@clusterhq.com>
+
+# Note: apt-get install requires the `-1` (epoch) suffix.
+ENV FLOCKER_VERSION 1.14.0-1
```

Comment:
it does not find 1.15.0, I probably need to specify the precise version of flocker.